libraries/libfc/src/io/json.cpp

```diff
@@ -284,14 +286,14 @@ namespace fc
               case '7':
               case '8':
               case '9':
-                 ss.put( in.get() );
+                 s += in.get();
                  break;
               case '\0':
                  FC_THROW_EXCEPTION( eof_exception, "unexpected end of file" );
               default:
                  if( isalnum( c ) )
                  {
-                    return ss.str() + stringFromToken( in );
+                    return s + stringFromToken( in );
```

Comment:
```suggestion
                    s += stringFromToken( in );
                    return s;
```

might be a little more efficielt

Reply:
Done.